doc/nrf/device_guides/working_with_nrf/nrf52/developing.rst

```diff
@@ -58,6 +58,10 @@ To enable support for FOTA updates, do the following:
    * SMP BT reassembly feature.
    * The :kconfig:option:`CONFIG_NCS_SAMPLE_MCUMGR_BT_OTA_DFU_SPEEDUP` Kconfig option automatically extends the Bluetooth buffers, which allows to speed up the FOTA transfer over Bluetooth, but also increases RAM usage.
 
+.. note::
+   The :kconfig:option:`CONFIG_NCS_SAMPLE_MCUMGR_BT_OTA_DFU` Kconfig option can be used with nRF52 Series devices to enable MCUmgr to perform firmware over-the-air (FOTA) updates using Bluetooth LE.
+   It can be used along with other samples, and is only meant for development and testing purposes.
```

Comment:
When I read this line as a user I think:
"Okay so you tell me about FOTA in docs, and then you say that I should not use this. Then what am I supposed to use instead for my application and release?!"

Reply:
Hm, good point. I don't think I can answer that fully myself, unfortunately.
I think all samples in NCS are meant to be for development and test purposes only - either way, SMEs wanted docs to reflect that this "common" code is only for those purposes.

Reply:
Here is what we tell users in [DevAcademy](https://academy.nordicsemi.com/courses/nrf-connect-sdk-intermediate/lessons/lesson-8-bootloaders-and-dfu-fota/topic/exercise-3-fota-over-bluetooth-low-energy/)
"In Zephyr, a set of configurations are needed to enable FOTA. An example of how to set this up can be seen in the [SMP Server sample](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/samples/subsys/mgmt/mcumgr/smp_svr/README.html). However, the nRF Connect SDK includes the option [CONFIG_NCS_SAMPLE_MCUMGR_BT_OTA_DFU](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/kconfig/index.html#CONFIG_NCS_SAMPLE_MCUMGR_BT_OTA_DFU), which sets all needed configurations for us. This option makes it easier for us to enable FOTA, so we will use that.
This configuration is named CONFIG_NCS_SAMPLE… as it makes some assumptions about the project. For your custom project, I recommend that you look at what configurations [CONFIG_NCS_SAMPLE_MCUMGR_BT_OTA_DFU](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/kconfig/index.html#CONFIG_NCS_SAMPLE_MCUMGR_BT_OTA_DFU) set for you, and include the ones from there you need instead."

 (well, I wrote it so this is me quoting myself as a source)

Reply:
I see your point.
This requires SME input/input @nvlsianpu @kapi-no - do we have any thoughts here on how the text in this PR can be reworded? Or should text from DevAcademy? My impression from our meeting was that this is only to be used for developing and testing, not final production.

Reply:
I think it can be used for production if the suggested configuration matches the user's requirements. The [CONFIG_NCS_SAMPLE_MCUMGR_BT_OTA_DFU](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/kconfig/index.html#CONFIG_NCS_SAMPLE_MCUMGR_BT_OTA_DFU) option only aggregates the necessary Kconfigs from sdk-nrf and sdk-zephyr. It allows the user to enable DFU functionality with a single option. I would suggest changing the last sentence as follows:

`It can be used along with other samples, and is meant as a demonstration of the default DFU configuration over Bluetooth.`

Please note that this sentence will also propagate to the nRF53 guide so mentioning nRF52 Series in the first sentence would confuse the nRF53 UG readers.

Reply:
Thanks, Kamil. Applied, and removed the "nRF52 Series" part of that note.